packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setConfig.ts

```diff
@@ -24,7 +26,82 @@ type ContractInput = [
   offchain_config: string[],
 ]
 
-const makeContractInput = async (input: SetConfigInput): Promise<ContractInput> => {
+const makeUserInput = async (flags, args, env): Promise<SetConfigInput> => {
+  if (flags.input) return flags.input as SetConfigInput
+  if (flags.rdd) {
```

Comment:
had to move the rdd part into `makeUserInput` to pass the input validation from gauntlet-core (cc @archseer)